app/assets/stylesheets/nonprofits/donation_form/title_row.scss

```diff
@@ -6,9 +6,10 @@
 	position: relative;
 	overflow: hidden;
 	background: $fog;
+	display: flex;
+	gap: 15px;
 }
 .titleRow-info {
```

Comment:
I think everything should be fine here but I just wanted to make sure you confirmed where these various styles are used throughout the codebase. I think they're only used on the donation_form but let's just say I've been surprised before :smile:

Reply:
@wwahammy Hmm, I am also seeing `titleRow` and `titleRow-info` in `change-amount-wizard.es6`. I'm not sure where to find that in the UI yet to investigate. It does look like it's basically doing the same thing there at first glance.

Reply:
Found that, it appears to look the same as before with this / be rendering essentially the same thing